### PR TITLE
[macos] Refactor framework skipping logic in mmp

### PR DESCRIPTION
**Pull request by chamons**
- Fixup comment in https://github.com/xamarin/xamarin-macios/pull/1968 that was missed

### CODE REVIEW

**Comment by chamons** [comment]
@spouliot since you requested it - https://github.com/xamarin/xamarin-macios/pull/1968#discussion_r110212265

File: tools/common/Assembly.cs [hunk 1]
```diff
@@ -405,6 +400,13 @@ public void ComputeLinkerFlags ()
 						break;
 #endif
 					default:
+#if MONOMAC
+
+						string path = Path.GetDirectoryName (name);
+						if (!path.StartsWith ("/System/Library/Frameworks"))
```
Comment (spouliot) [inline tools/common/Assembly.cs]: add `StringComparison.Ordinal`

File: tools/common/Assembly.cs [hunk 2]
```diff
@@ -405,6 +400,13 @@ public void ComputeLinkerFlags ()
 						break;
 #endif
 					default:
+#if MONOMAC
+
```
Comment (spouliot) [inline tools/common/Assembly.cs]: minor: extra empty line

File: tools/common/Assembly.cs (hunk 1 above)
Comment (chamons) [inline tools/common/Assembly.cs]: Fixed.

**Pull request merged**